DfciPkg/DfciPkg.dsc

```diff
@@ -92,14 +92,6 @@
   RngLib|MdePkg/Library/BaseRngLib/BaseRngLib.inf
   VariablePolicyHelperLib|MdeModulePkg/Library/VariablePolicyHelperLib/VariablePolicyHelperLib.inf
 
-!if $(TOOL_CHAIN_TAG) == VS2017 or $(TOOL_CHAIN_TAG) == VS2015 or $(TOOL_CHAIN_TAG) == VS2019 or $(TOOL_CHAIN_TAG) == VS2022

```

Comment:
Simply removing this means no further change is required/coming?

Reply:
Discussed offline. They're coming in through `MdePkg/MdeLibs.dsc.inc` in this case.

Reply:
This DSC includes MdePkg/MdeLibs.dsc.inc which has the stack cookie library defs.